src/concrete/ml/common/utils.py

```diff
@@ -582,3 +583,19 @@ def all_values_are_of_dtype(*values: Any, dtypes: Union[str, List[str]]) -> bool
         supported_dtypes[dtype] = supported_dtype
 
     return all(_is_of_dtype(value, supported_dtypes) for value in values)
+
+
+def force_auto_adjust_rounder_in_configuration(configuration, **kwargs):
```

Comment:
also maybe type the configuration parameter + add the type in the docstring (surprise mape pcc didn't raise any issue here)

Reply:
(I think you forgot this change !)